### ISSUE TITLE
Problem with msn.com accounts: Unable to authenticate: Unknown error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#312]
```
What steps will reproduce the problem?
1. Try to logging using hotmail.com its ok 
2. Try to logging using msn.com fail
3. Error : Unable to authenticate: Unknown error

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
pidgin-2.8.0
msn-pecan-0.1.2

Please provide any additional information below.

```

Original issue reported on code.google.com by `ingen...@gmail.com` on 18 Jun 2011 at 2:47

**Comment by GoogleCodeExporter** [comment]
```
here, hv same bug. can't login 

Unable to authenticate: nexus stream error
```

Original comment by `H.alz3abi@gmail.com` on 13 Jul 2011 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
How can I get a msn.com account?
```

Original comment by `felipe.contreras` on 15 Jul 2011 at 9:03
- Changed title: **Problem with msn.com accounts: Unable to authenticate: Unknown error**
- Changed state: **Accepted**
- Added labels: **Type-Defect**, **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
You can get a msn account here:
https://accountservices.passport.net/reg.srf?ns=msn.com&sl=1&vv=1000&mkt=ES-ES&l
c=3082
```

Original comment by `Littlefo...@gmail.com` on 1 Aug 2011 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
Thanks. I could create the account, but had some trouble fixing the issue. I 
think it's done, I just need to clean it up :)
```

Original comment by `felipe.contreras` on 4 Aug 2011 at 4:44
- Changed state: **Started**
- Added labels: **Login**

**Comment by GoogleCodeExporter** [comment]
```
Hmm, so the issue is that msn.com accounts need redirection, which in theory 
could happen with any kind of account, I mostly have the code ready, but now I 
can't really tested because my account is not redirecting any more.

Can you verify that sometimes the current code works fine?
```

Original comment by `felipe.contreras` on 6 Aug 2011 at 10:42

**Comment by GoogleCodeExporter** [comment]
```
Also receiving "Unable to authenticate: nexus stream error".

Windows 7 64-bit
Pidgin 2.10.0
msn-pecan 0.1.3

```

Original comment by `mikerjoh...@gmail.com` on 23 Aug 2011 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
@mikerjoh That's not possible... Nexus is not used in 0.1.3 =/

Can you make sure? Try to uninstall msn-pecan, see that libmsn-pecan.dll is not 
the directory of Pidgin plugins, and then try again.
```

Original comment by `felipe.contreras` on 23 Aug 2011 at 3:37

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your reply!  You've found my problem.  I had put a 1.0 version of 
the libmsn-pecan.dll in my %APPDATA%\Roaming\.purple\plugins folder long ago 
and forgot about it.  Seems this was overriding the one installed in the 
application directory.  Thanks for your help!
```

Original comment by `mikerjoh...@gmail.com` on 23 Aug 2011 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
@mikerjoh Cool! Is this with a msn.com account as the bug report summary 
mentions?
```

Original comment by `felipe.contreras` on 23 Aug 2011 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
Eh - missed that part too.  It's just a Windows Live account using my gmail 
address. Sorry for the false alarm.  Searched for the nexus stream error and 
found this issue.
```

Original comment by `mikerjoh...@gmail.com` on 23 Aug 2011 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
Ok, I'm looking for somebody that can reproduce this issue, because my msn.com 
account suddenly just started to work perfectly fine, and I can't test the code 
I had in progress any more :(

I'll guess I'll just add some unit testing for the new functions, and hope for 
the best.
```

Original comment by `felipe.contreras` on 23 Aug 2011 at 5:29

**Comment by GoogleCodeExporter** [comment]
```
I still have the problem with the msn account, under ubuntu natty and Windows 
7. How can I help?.
```

Original comment by `Littlefo...@gmail.com` on 8 Sep 2011 at 9:17

Attachments:
- [Doc 1](https://storage.googleapis.com/google-code-attachments/msn-pecan/issue-312/comment-12/Doc 1)

**Comment by GoogleCodeExporter** [comment]
```
sorry, I don't attach the debug log correctly
```

Original comment by `Littlefo...@gmail.com` on 11 Sep 2011 at 3:33

Attachments:
- [debug-log.txt](https://storage.googleapis.com/google-code-attachments/msn-pecan/issue-312/comment-13/debug-log.txt)

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem on Pidgin 2.7.11 (libpurple 2.7.11) on Linux mint, 
using msn-pecan 0.1.1-1 (installed via synaptic)

When I try to login it says: "No se pudo autentificar: error de transmisión en 
nexus" (unable to autenticate, nexus error).
Relevant note: my windows live account is not @hotmail.com .... its my very own 
private email address (it works on MSN and on pidgin regular MSN).

MSN works with pidgin in my linux mint PC, but custom emoticons don't, and 
someone recommended me as a "workaround" to try msn-pecan to solve the 
emoticons problem.
```

Original comment by `vladimir...@gmail.com` on 6 Oct 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
@14: your issue is fixed in msn-pecan >= 0.1.2
```

Original comment by `devid...@gmail.com` on 15 Oct 2011 at 1:11

**Comment by GoogleCodeExporter** [comment]
```
Yeah please try a newer version of msn-pecan and verify that the fix works.
```

Original comment by `felipe.contreras` on 23 Feb 2012 at 8:33
- Changed state: **Fixed**
